.github/workflows/jdk17.yml

```diff
@@ -1,4 +1,4 @@
-name: Java 11 CI with Maven
+name: Java 17 CI with Maven
```

Comment:
Not critical, but you could rename the yml file so it doesn't require a name change with each JDK change.

Reply:
okay!